cuqipy_fenics/pde.py

```diff
@@ -106,22 +226,104 @@ def gradient_wrt_parameter(self):
     def _create_observation_operator(self, observation_operator):
         raise NotImplementedError
 
+    def _is_parameter_updated(self, value):
```

Comment:
Can you explain this method more clearly. It takes a value as input, which it checks against the existing parameter values. The docstring should explain this for future reference.

Reply:
Good point, I ended up also changing the method name and the input name for clarity, in addition to updating the documentation.